core/server.py

```
@@ -196,6 +196,32 @@ async def start_google_auth(
 
     logger.info(f"Tool 'start_google_auth' invoked for user_google_email: '{user_google_email}', service: '{service_name}', session: '{mcp_session_id}'.")
 
+    # Check if we're using access token mode
+    access_token = os.getenv('GOOGLE_OAUTH_ACCESS_TOKEN')
+    if access_token:
+        logger.info(f"[start_google_auth] Access token mode detected - OAuth management is disabled")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)